Adadie/Fresh
# Integration

#### What does this PR do?
This PR will update styles of all the pages and integrate the frontend admin section with the backend.

#### Description of Task to be completed?

- Change padding and margin of all web pages.
- Increase comments input box size.
- Increase media query screen size.
- Integrate the login page, sign up page, post form after login, and render posted articles in the Posted articles section.
#### How should this be manually tested?
- Clone this repository in your local directory [link](https://github.com/Junior-Adadie/Fresh.git).
- Open index.html with your browser.
- Create an account in the signup page.
- Login with the created account.
- Create an article.
- Check if it's rendered in the posted article section.
- Delete the article and verify if it's deleted.
- Update the article and verify if it's updated.
- Check if the comments input box height is higher than the message box.
- Make sure your browser is updated.
- Check if no excess or unnecessary padding or margin is on any page.

#### Any background context you want to provide?
 The top padding of every page was 200px and the media query screen size was 768Px.

#### What are the relevant Trello stories?
[Trello Card](https://trello.com/c/30WoVlEU/19-change-display-styles)

#### Screenshots (if appropriate)
![Capture2](https://user-images.githubusercontent.com/68109866/88719039-52bae380-d12b-11ea-9306-166114fff281.PNG)
![Capture](https://user-images.githubusercontent.com/68109866/88719033-5189b680-d12b-11ea-871f-0cfc07e9198a.PNG)
